src/api/applicant/content-types/applicant/lifecycles.js

```diff
@@ -1,14 +1,54 @@
+const _ = require('lodash');
```

Comment:
How are we using loadash?

Reply:
We're using it here https://github.com/dev-launchers/strapiv4/pull/22/files#diff-6144805fbeed9ef42cece8f4c59076e25b68343b142ce530301ffd522cc6add2R41

`var relevantInfo = _.omit(event.params.data, ['email', 'age', 'zip', 'createdBy', 'updatedBy', 'createdAt', 'updatedAt', 'publishedAt']);`

It's a bit confusing with the underscore, I'll spell out `lodash` instead for readability